### PR TITLE
made dashboard responsive

### PR DESCRIPTION
**Pull request by swatified**
Hey I made dashboard responsive as reviewed by @Shobhit-Nagpal and @sudhanshutech 

https://github.com/user-attachments/assets/a37dd550-42b0-4b64-85c8-cbfd183dcfba

You can update the earlier level or assign this level1 if you dont mind.

### CODE REVIEW

**Comment by swatified** [comment]
@himeshparashar have a look. this is under #86

File: app/dashboard/page.tsx [hunk 1]
```diff
@@ -14,71 +15,133 @@ import {
...
+  
+      try {
+        if (!apiKey) {
+          throw new Error("Gemini API key is not set. Please check your environment variables.");
+        }
+  
+        // Combine the static prompt with user input
+        const fullPrompt = `${customPrompt} ${inputText}`;
+  
+        console.log("Generating content with prompt:", fullPrompt);
```
Comment (sudhanshutech) [inline app/dashboard/page.tsx:106]: remove the console logs please

**Review by sudhanshutech — CHANGES_REQUESTED** [review]

**Comment by sudhanshutech** [comment]
looks good to me . good work @swatified

**Comment by swatified** [comment]
@himeshparashar please check once :(

My two issues got merged in single pr somehow. i didnt even raise for gemini text generation yet. but level 2?? for dashboard responsive.. then what about my gemini one? consider once :') cz it cant be recommited now na. only levels can be changed. either increase here or give here for gemini and dashboard responsive one's can go int dashboard ui fix i made? can update level there or sum? @himeshparashar hope im making sense

**Comment by swatified** [comment]
i mean issue #54 and #86 got merged in this pr sadly :') @himeshparashar

**Comment by himeshparashar** [comment]
@swatified please use branching to avoid this type of situation create a new branch for every new issue 

for now I have changed the title considering for both responsive and gemini generation 

and will add level 2 for now 

and for the gemini generation I have suggested a popup modal if you wish to work on that 
then that will make a another new pr I will add level 1 or 2 accordingly 

does this solves this ? (actually very confusion to me also 🥲)

**Comment by swatified** [comment]
I asked for more levels because of two owkrs getting mixed actually? 😔 but it got down to from 2 to 1 now lol. Yes I'm working on that way rn. Sorry for creating chaos. I apologize.

**Comment by himeshparashar** [comment]
@swatified will give you lvl 3 on the other pr for the popup part

**Comment by swatified** [comment]
@himeshparashar okie but am doing it as redirection to other page tho. not pop-up way

**Comment by himeshparashar** [comment]
@swatified ok

**Pull request merged**